query-engine/prisma/src/main.rs

```diff
@@ -115,17 +125,33 @@ async fn main() -> Result<(), AnyError> {
     } else {
         init_logger()?;
 
+        let default_host = [127, 0, 0, 1];
+        let default_port = 4466;
+
         let port = matches
             .value_of("port")
             .map(|p| p.to_owned())
             .or_else(|| env::var("PORT").ok())
             .and_then(|p| p.parse::<u16>().ok())
-            .unwrap_or_else(|| 4466);
+            .unwrap_or_else(|| default_port);
+
+        let host = matches
+            .value_of("host")
+            .map(|p| p.to_owned())
+            .or_else(|| env::var("HOST").ok())
+            .and_then(|p| Some(IpAddr::from_str(&p).expect("Invalid Host provided")))
+            .unwrap_or_else(|| IpAddr::from(default_host));
+
+        if !host.is_ipv4() {
```

Comment:
Why this?

https://doc.rust-lang.org/std/net/enum.IpAddr.html

Enum can be either 4 or 6. Did you have some other reason on forcing ipv4?

Reply:
No reason. It should JUST WORK. The original interface was worse than this.

Reply:
Yep, it's already removed :) It does indeed just work